src/IO/lbsLoadReaderVT.py

```diff
@@ -202,18 +139,56 @@ def read_iteration(self, n_p, time_step):
         # Iterate over all processors
         for p in range(n_p):
             # Read data for given iteration and assign it to processor
-            proc_iter_map = self.read(p, time_step)
+            proc_iter_map = self.read(p, phase_id)
 
             # Try to retrieve processor information at given time-step
             try:
-                procs[p] = proc_iter_map[time_step]
+                procs[p] = proc_iter_map[phase_id]
             except KeyError:
                 print(bcolors.ERR
-                      + "*  ERROR: [LoadReaderVT] Could not retrieve information for processor {} at time_step {}".format(
+                      + "*  ERROR: [LoadReaderVT] Could not retrieve information for processor {} at phase_id {}".format(
                     p,
-                    time_step)
+                    phase_id)
                       + bcolors.END)
                 sys.exit(1)
 
         # Return populated list of processors
         return procs
+
+    def json_reader(self, returned_dict: dict, file_name: str, phase_ids, node_id: int) -> dict:
+        """ Reader compatible with current VT Object Map files (json)
+        """
+        with open(file_name, 'rb') as compr_json_file:
+            compr_bytes = compr_json_file.read()
+            try:
+                decompr_bytes = brotli.decompress(compr_bytes)
+                decompressed_dict = json.loads(decompr_bytes.decode('utf-8'))
+            except brotli.error:
+                decompressed_dict = json.loads(compr_bytes.decode('utf-8'))
+
+        # defining phases from file
+        phases = decompressed_dict['phases']
+        # iterating over phases
+        for phase in phases:
+            phase_id = phase['id']
+            for task in phase['tasks']:
+                task_time = task.get('time')
+                task_object_id = task.get('entity').get('id')
```

Comment:
Do we need to verify that the entity type is in fact `object`?

Reply:
There is a schema validator coming soon and we might need to have a quick chat regarding what else could it be.